satpy/composites/__init__.py

```diff
@@ -313,17 +313,30 @@ class SunZenithCorrectorBase(CompositeBase):
 
     coszen = WeakValueDictionary()
 
+    def __init__(self, min_sza=0.0, max_sza=95.0, **kwargs):
```

Comment:
I think `min_sza` should be remove, it isn't something that is used in practice.